src/Response/Struct/ItemOperatingHours.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\InpostBundle\Response\Struct;
+
+class ItemOperatingHours
+{
+    public array $sunday = [];
+    public array $monday = [];
+    public array $tuesday = [];
+    public array $wednesday = [];
+    public array $thursday = [];
+    public array $friday = [];
+    public array $saturday = [];
+
+    public static function fromArray(array $operatingHours): self
+    {
+        $self = new self();
+
+        if (!isset($operatingHours['customer'])) {
+            return $self;
+        }
+
+        if (!is_array($operatingHours['customer'])) {
+            return $self;
+        }
+
+        foreach ($operatingHours['customer'] as $day => $hours) {
+            if (isset($self->{strtolower($day)})) {
```

Comment:
Please note that if the partner changes the API or naming convention, the data will be invalid or not parsed correctly.

Reply:
This is a phenomenon that applies to every integration in the world and to every field in the response API, not just this specific case. For this reason, the partner provides us with a versioned API. So we trust the partner and at the same time we do not display this field if it is empty, verifying carefully a few lines back.

Reply:
Ok, but you have hardcoded the names of the days of the week. 
Be careful with this  because it is slippery: `$self->{strtolower($day)}`.